docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+---
+title: Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID
+---
+
+In this guide, you will learn how to create host certificates through Machine ID for OpenSSH servers. This allows for the benefits of
+Machine ID with OpenSSH nodes and reduces risk by ensuring that short-lived certificates issued to tbot users on Teleport clusters can be successfully applied.
```

Comment:
Teleport issues both user certs and host certs. This guide is about host certs, so I find the "certificates issued to tbot _users_" part a little confusing.